src/main/java/com/skalashynski/spring/springboot/web/controller/StudentController.java

```diff
@@ -64,4 +64,10 @@ public void delete(@PathVariable Long id) {
     public List<Student> getByName(@RequestParam Map<String, String> allParams) {
         return studentService.findByFirstName(allParams.get("name"));
     }
+
+    @GetMapping("/search/by/birthdays")
+    public List<Student>getByBirthdaysDates(@RequestParam Map<String, String> allParams){
+        return studentService.findBetweenBirthdays(java.sql.Date.valueOf(allParams.get("from")),
+            java.sql.Date.valueOf(allParams.get("to")));
+    }
```

Comment:
Dangerous code. What if allParams.get () return null?
Try to code and cover testing again this piece